kombu/transport/redis.py

```diff
@@ -1066,14 +1066,17 @@ class SentinelChannel(Channel):
     from_transport_options = Channel.from_transport_options + (
         'master_name',
         'min_other_sentinels',
-        'sentinel_kwargs')
+        'sentinel_kwargs',
+        'connection_kwargs')
 
     connection_class = sentinel.SentinelManagedConnection if sentinel else None
 
     def _sentinel_managed_pool(self, async=False):
         connparams = self._connparams(async)
 
-        additional_params = connparams.copy()
+        additional_params = getattr(self,
+                                    'connection_kwargs',
+                                    connparams.copy())
```

Comment:
Of course, but I propose to wait a little before updating the documentation.
A few days ago, I sent a request for adding sentinel support for the backend:
https://github.com/celery/celery/pull/4053
The problem is that my backend code is not very good. I hope that someone
from the community will do it better than I, or tell me how to rewrite it, but the code for the kombu is not bad).